src/MICore/project.json

```diff
@@ -1,25 +1,42 @@
 {
-  "supports": {
-    "net46.app": {},
-    "dnxcore50.app": {}
-  },
   "dependencies": {
-    "MicroBuild.Core": "0.2.0",
-    "Microsoft.NETCore": "5.0.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
-    "System.Diagnostics.Process": "4.0.0-beta-23225",
-    "System.Net.Security": "4.0.0-beta-23225",
-    "System.Net.Sockets": "4.1.0-beta-23225",
-    "System.Reflection.TypeExtensions": "4.0.0",
-    "System.Security.Cryptography.X509Certificates": "4.0.0-beta-23225",
-    "System.Threading.Thread": "4.0.0-beta-23225",
-    "System.Xml.XmlSerializer": "4.0.11-beta-23225",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-beta-23225",
-    "System.IO.FileSystem.Watcher": "4.0.0-beta-23225"
+    "Microsoft.VisualStudio.Debugger.Interop.Portable": "1.0.1",
```

Comment:
Or did you just make most of the files the same so that it was easier to maintain?